Sources/TuistDependencies/Mappers/ExternalDependencyPathWorkspaceMapper.swift

```diff
@@ -33,7 +33,7 @@ public final class ExternalDependencyPathWorkspaceMapper: WorkspaceMapping {
         var base = project.settings.base
         // Keep the value if already defined
         if base["SRCROOT"] == nil {
-            base["SRCROOT"] = SettingValue(stringLiteral: project.sourceRootPath.pathString)
+            base["SRCROOT"] = SettingValue(stringLiteral: project.sourceRootPath.relative(to: project.path).pathString)
```

Comment:
is this actually correct? there are several acceptance tests failing – when the xcodeproj is in a different path, should this be relative to `xcodeProjPath`?

Reply:
> is this actually correct? there are several acceptance tests failing – when the xcodeproj is in a different path, should this be relative to xcodeProjPath?

Yes :)